src/app/clarin-licenses/clarin-license-table/clarin-license-table.component.html

```diff
@@ -22,32 +22,33 @@
                      [retainScrollPosition]="true">
         <table class="table table-striped table-bordered align-middle my-2">
           <thead>
-          <tr class="col-12">
-            <th class="col-1" style="width: 3.5%; flex: 0 0 3.5%;max-width: 3.5%;"></th>
-            <th class="col-3">{{"clarin-license.table.name" | translate}}</th>
-            <th class="col-2">{{"clarin-license.table.definition" | translate}}</th>
+          <tr>
+            <th style="width: 3.5%; max-width: 3.5%;"></th>
+            <th style="width: 25%;">{{"clarin-license.table.name" | translate}}</th>
```

Comment:
we are going with this kind of width style?
does not seem too flexible

Reply:
https://github.com/search?q=repo%3Adataquest-dev%2Fdspace-angular+%2Fstyle%3D%22.*width%3A%2F&type=code

Reply:
fixed